modules/table/src/test/java/org/apache/ignite/internal/table/type/NumericTypesSerializerTest.java

```diff
@@ -290,33 +220,6 @@ public void testDecimalScaleTooLarge(int scale, String message) {
         );
     }
 
-    /**
-     * Test.
-     */
-    @ParameterizedTest
-    @MethodSource("sameDecimals")
-    public void testSameBinaryRepresentation(Pair<BigInteger, BigInteger> pair) {
```

Comment:
Why have you removed the test?

Reply:
What is a type that corresponds to BigInteger now?

Reply:
Fixed. I was confused by a type signature, that is not correct it is a pair of BigDecimals.